x-pack/plugins/screenshotting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -63,11 +62,6 @@ interface ClosePageResult {
   metrics?: PerformanceMetrics;
 }
 
-export const DEFAULT_VIEWPORT = {
```

Comment:
I refined this object more to align to what Puppeteer expects, and moved it to the `layouts` directory of this plugin